src/pages/auth.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
We should be able to remove these lines, all that needs to happen is to add the keyword `void` on promises